kubectl-plugin/pkg/cmd/cluster/get_test.go

```diff
@@ -0,0 +1,195 @@
+package cluster
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/gosuri/uitable"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	fakedynamic "k8s.io/client-go/dynamic/fake"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
+	cmdtesting "k8s.io/kubectl/pkg/cmd/testing"
+)
+
+func TestRayClusterGetComplete(t *testing.T) {
```

Comment:
Can you add some comments to this test? It is not very straightforward to me. I understand the test after I trace the code.